src/main/webapp/WEB-INF/views/manufacturer/allManufacturer.jsp

```diff
@@ -0,0 +1,22 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Title</title>
+</head>
+<body>
+<table>
+<td>Id</td>
+<td>Name</td>
+<td>Country</td>
+<c:forEach items="${manufacturers}" var="manufacturer">
+  <tr>
+    <td><c:out value="${manufacturer.id}"/></td>
+    <td><c:out value="${manufacturer.name}"/></td>
+    <td><c:out value="${manufacturer.country}"/></td>
+    <td><a href="${pageContext.request.contextPath}/manufacturers/delete?id=${manufacturer.id}">Delete</a> </td>
+  </tr>
+</c:forEach>
+</table>
```

Comment:
please also pay attention to the alignment issues of your jsps. You can try to use ctrl+alt+L for auto alignment